test/downstream/HybridNODE.jl

```diff
@@ -112,6 +113,85 @@ function test_hybridNODE2(sensealg)
     println("  ")
 end
 
+mutable struct Affect{T}
+      callback_data::T
+end
+compute_index(t) = Int(t ÷ 1 + 1)
+Zygote.@nograd compute_index #avoid taking grads of this function
+function (cb::Affect)(integrator)
+    indx = compute_index(integrator.t)
```

Comment:
The test failure is related to the index assignment (`t ÷ 1`). I had hoped that `nograd` could handle this. 
```
MethodError: no method matching div(::ReverseDiff.TrackedReal{Float32, Float32, ReverseDiff.TrackedArray{Float32, Float32, 1, Vector{Float32}, Vector{Float32}}}, ::ReverseDiff.TrackedReal{Float32, Float32, ReverseDiff.TrackedArray{Float32, Float32, 1, Vector{Float32}, Vector{Float32}}}, ::RoundingMode{:ToZero})
```

Reply:
`round(Int,t)+1`?